src/commands/remoteDebug/startRemoteDebug.ts

```diff
@@ -56,7 +56,7 @@ async function startRemoteDebugInternal(actionContext: IActionContext, node?: Si
         await callWithTelemetryAndErrorHandling('appService.remoteDebugStartProxy', async function (this: IActionContext): Promise<void> {
             this.suppressErrorDisplay = true;
             this.rethrowError = true;
-            await tunnelProxy.startProxy();
+            await tunnelProxy.startProxy(49494);

```

Comment:
Why 49494?

Reply:
It's the port that's used for remote debugging (as far as I can tell) This seems dependent on kudu (we never set this port for remote debugging, it just happened to be the port that was opened)